cost/budget_alerts/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 # Changelog
-# v1.16
+
+## v1.16
 
 - Now using OAuth2 credentials instead of the built-in Rightscale authentication
 
```

Comment:
There are 2 v1.15 entries. Make the rs_optima_host one v1.16, then bump up the latest to v1.17.